### ISSUE TITLE
Incorrect path in TgzRelativePath

### ISSUE THREAD

**Issue by g4s8** [artipie/npm-adapter#169]
After the NPM uploads a package without scope, the package cannot be downloaded based on the tarball.                                   

Code to reproduce:                                                                                                                                        
```                                                                                                                                               
 new TgzRelativePath(                                                                                                                   
            "http://some.host/artifactory/api/npm/npm-tmptest/zlComponent/1.0.1/zlComponent-1.0.1.tgz")       
            .relative()                                                                                                                 
```
                                                                                                                                     
expected: `zlComponent/1.0.1/zlComponent-1.0.1.tgz`
result: `1.0.1/zlComponent-1.0.1.tgz`

**Comment by olenagerasimova** [comment]
@g4s8 I do not know nodejs very well, can you please provide example package?

**Comment by kirill-h5i** [comment]
@olenagerasimova please check example archive:
[jQuery-1.7.4.tar.gz](https://github.com/artipie/npm-adapter/files/5873052/jQuery-1.7.4.tar.gz)

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) I see that you unassigned this issue; the order is still assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) though; to cancel the order use `refuse`, as in [§6](http://www.zerocracy.com/policy.html#6)

**Comment by genryxy** [comment]
@g4s8 I do not think that expected result is correct as we have path to tgz archive: `package-name/-/package-name-1.0.0.tgz` and path to index file (`meta.json`): `package-name/meta.json`. 
Besides, registry also has a similar structure, for example, https://registry.npmjs.org/asdas/-/asdas-1.0.0.tgz

**Comment by dingdaoyi** [comment]
For historical reasons, packages uploaded by non-npm clients have a version number in their upload path, such as Jfrog ArtiFactory

**Comment by 0crat** [comment]
Job `gh:artipie/npm-adapter#169` is not assigned, can't get performer

**Comment by g4s8** [comment]
@rultor release, tag=`0.9`

**Comment by g4s8** [comment]
@genryxy it should be released soon `0.9`
